tests/30-kubernetes-e2e-check.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
```

Comment:
I have heard `#!/usr/bin/env python3` is more portable. This is not a blocking comment, just want to be consistent with other stuff I have seen out of the python tools.